python/cudf/cudf/io/parquet.py

```diff
@@ -8,6 +8,7 @@
 
 import cudf
 import cudf._lib.parquet as libparquet
+from cudf._lib.parquet import ParquetChunkedWriter  # noqa
```

Comment:
Instead of doing this could you just add a `ParquetChunkedWriter = libparquet.ParquetChunkedWriter` in the code?

Reply:
done